scripts/sysman/virtualFunctionManagement.yml

```diff
@@ -0,0 +1,252 @@
+class: $sVFManagement
+name: GetVFMemoryUtilizationExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_vf_handle_t
+      name: hVFhandle
+      desc: "[in] Handle for the component."
+    - type: "uint32_t*"
+      name: pCount
+      desc: |
+            [in,out] Pointer to the number of VF memory stats descriptors.
+              - if count is zero, the driver shall update the value with the total number of memory stats available.
+              - if count is greater than the total number of memory stats available, the driver shall update the value with the correct number of memory stats available.
+              - The count returned is the sum of number of VF instances currently available and the PF instance.
+    - type: $s_vf_util_mem_exp_t*
+      name: pMemUtil
+      desc: |
+            [in,out][optional][range(0, *pCount)] array of memory group activity counters.
+              - if count is less than the total number of memory stats available, then driver shall only retrieve that number of stats.
+              - the implementation shall populate the vector pCount-1 number of VF memory stats.
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get engine activity stats for each available engine group associated with Virtual Function (VF)"
+class: $sVFManagement
+name: GetVFEngineUtilizationExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_vf_handle_t
+      name: hVFhandle
+      desc: "[in] Handle for the component."
+    - type: "uint32_t*"
+      name: pCount
+      desc: |
+            [in,out] Pointer to the number of VF engine stats descriptors.
+              - if count is zero, the driver shall update the value with the total number of engine stats available.
+              - if count is greater than the total number of engine stats available, the driver shall update the value with the correct number of engine stats available.
+              - The count returned is the sum of number of VF instances currently available and the PF instance.
+    - type: $s_vf_util_engine_exp_t*
+      name: pMemUtil
+      desc: |
+            [in,out][optional][range(0, *pCount)] array of engine group activity counters.
+              - if count is less than the total number of engine stats available, then driver shall only retrieve that number of stats.
+              - the implementation shall populate the vector pCount-1 number of VF engine stats.
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Configure utilization telemetry enabled or disabled associated with Virtual Function (VF)"
```

Comment:
Clarification: Does this disable querying telemtery from inside the VF ?
Please share some use-case of this.